src/components/editor/editor-ui/bottomTab/preview-stage.tsx

```
@@ -0,0 +1,86 @@
+              [ElEMENT_TYPE.UPLOAD]: (
+                <PreviewUploadCanvas element={el as UploadElement} />
+              ),
+              [ElEMENT_TYPE.IMAGE]: (
+                <PreviewImageCanvas element={el as ImageElement} />
+              ),
+              [ElEMENT_TYPE.QR]: <PreviewQrCanvas element={el as QrElement} />,
+              [ElEMENT_TYPE.SOCIAL]: (
+                <PreviewSocialCanvas element={el as SocialElement} />
+              ),
+            }}
+            default={<></>}
```

Biome (1.9.4)

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.

(lint/complexity/noUselessFragments)